UPGRADE-3.x.md

```diff
@@ -9,3 +9,19 @@ UPGRADE FROM 3.0 to 3.1
 All files under the ``Tests`` directory are now correctly handled as internal test classes. 
 You can't extend them anymore, because they are only loaded when running internal tests. 
 More information can be found in the [composer docs](https://getcomposer.org/doc/04-schema.md#autoload-dev).
+
+### Deprecated
+
+`$container` property in `Security/SessionDownloadStrategy` is deprecated. Use `SessionInterface` `$session` instead.
+
+Before:
+
+```php
+    $downloadStrategy = new SessionDownloadStrategy($translator, $container, $times);
```

Comment:
hmm, I am missing the typehint here, maybe its better so show the __construct() method with their typehints here to see the difference? @greg0ire